cnd/src/swap_protocols/swap_id.rs

```diff
@@ -50,6 +50,13 @@ impl fmt::Display for LocalSwapId {
 #[derive(Serialize, Deserialize, Clone, Copy, Debug, Eq, Hash, PartialEq)]
 pub struct SharedSwapId(Uuid);
 
+impl SharedSwapId {
+    /// Construct an all zero swap id (for testing).
+    pub fn nil() -> Self {
+        SharedSwapId(Uuid::nil())
+    }
+}
```

Comment:
```suggestion
#[cfg(test)]
impl SharedSwapId {
    /// Construct an all zero swap id (for testing).
    pub fn nil() -> Self {
        SharedSwapId(Uuid::nil())
    }
}
```

Don't push to prod code if it is only for test.

Reply:
Yes, thanks good idea - schooled me too.  Cheers man.